hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -1137,11 +1149,11 @@
           name: "rdfifo_notempty"
           desc: "If 1, the TPM_READ_FIFO still has pending data."
         } // f: rdfifo_notempty
-        { bits: "6:4"
+        { bits: "3+TpmRdFifoPtrW:4"
           name: "rdfifo_depth"
           desc: "This field represents the current read FIFO depth"
         } // f: rdfifo_depth
-        { bits: "10:8"
+        { bits: "7+TpmWrFifoPtrW:8"
```

Comment:
Would it make sense to calculate the offset based on `TpmRdFifoPtrW` here?
I.e., `4 + TpmRdFifoPtrW + TpmWrFifoPtrW : 5 + TpmRdFifoPtrW`?

Reply:
Actually, the bit offset I aligned to Hex digit explicitly. Then when memory dumped, the value is easily readable.